documentation/STRUCTURE.md

```diff
@@ -0,0 +1,161 @@
+
+Of our current notebooks the following would be most closely aligned with this section:
+
+- [Binary Classification on COMPAS dataset](https://github.com/fairlearn/fairlearn/blob/master/notebooks/Binary%20Classification%20on%20COMPAS%20dataset.ipynb) - although we should perhaps consider removing it since it may not do justice to this complex setup
+- [Binary Classification with the UCI Credit-card Default Dataset](https://github.com/fairlearn/fairlearn/blob/master/notebooks/Binary%20Classification%20with%20the%20UCI%20Credit-card%20Default%20Dataset.ipynb)
+- [Mitigating Disparities in Ranking from Binary Data](https://github.com/fairlearn/fairlearn/blob/master/notebooks/Mitigating%20Disparities%20in%20Ranking%20from%20Binary%20Data.ipynb)
+
+### Community
+
+This page should provide basic information about how to reach out in order to ask questions, provide feedback, report bugs or request features. Similarly, it should be clear how people can reach out if they want to contribute to Fairlearn, and where they can find small items to get started. [This](https://scikit-learn.org/dev/developers/contributing.html) is an example of how scikit-learn handles it through a contribution guide that is somewhat similar to ours that we currently have in the repo.
+
+We could also write up the history of the project, its roadmap, and how we envision governance to work. The code of conduct should certainly be mentioned as well.
+
+Some projects have a page showing the maintainers as well:
+
+- [scikit-learn](https://scikit-learn.org/stable/about.html#people)
+- [pandas](https://pandas.pydata.org/about/team.html)
+
+## Required steps
+
+To properly format math, text, and links across the pages we need to use ReST. Some of our existing documentation may have to be converted from Markdown to ReST, e.g. the terminology page.
+
+We need to find a way to present the dashboard in a website where it can't be interactive. Perhaps with screenshots for the user guides, but the case studies are downloadable as Jupyter notebooks. [Could we perhaps pre-calculate all metrics and show the interactive dashboard in the case studies?]
+
+We already have the examples gallery thanks to Adrin's work on sphinx-gallery: https://fairlearn.readthedocs.io/en/latest/index.html
+There will be plenty of work to
+
+- convert existing notebooks to case studies (or user guides)
+- write all user guides
+- write the installation guide
+
+The repository structure should look similar to [what scikit-learn has](https://github.com/scikit-learn/scikit-learn/tree/master/doc):
+
+- top-level doc directory has all the ReST files for the webpage except for API documentation and case studies
+- API documentation comes directly from the code documentation
+- case studies live in a separate top-level directory (scikit-learn calls it `examples`) as python files
+
+Putting it all together:
+
+- deploy through fairlearn.org
```

Comment:
Do we have a plan for that hosting? I know it's not strictly part of this document, but it is sort of important....

Reply:
it can be hosted on github.io, but accessible through fairlearn.org, if that's an option.

Reply:
It turns out we have the domain and a few others, but for now fairlearn.github.io seemed to work the easiest. We can revisit whether we want fairlearn.org

Reply:
I would recommend directing the `fairlearn.github.io` to `fairlearn.org` through a `CNAME`. Here's how: https://help.github.com/en/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site